src/sider/manglendemedvirkning/ikkeaktuell/IkkeAktuellSkjema.tsx

```diff
@@ -98,6 +100,14 @@ export default function IkkeAktuellSkjema() {
           <Button loading={sendVurdering.isPending} type="submit">
             {texts.buttons.save}
           </Button>
+          <Forhandsvisning
+            contentLabel={texts.buttons.previewContentLabel}
+            getDocumentComponents={() =>
+              getIkkeAktuellDocument({
+                begrunnelse: watch("begrunnelse"),
```

Comment:
Blir det mer riktig med `getValues`? 😊 Vet ikke om vi risikerer å re-rendre ettersom `watch(…)` hele tiden vil oppdatere seg hver gang vi endrer begrunnelsen🤔 Uansett ønsker vi vel kanskje å bare hente ut verdien i øyeblikket vi trykker på knappen
```suggestion
     begrunnelse: getValues("begrunnelse"),
```

Reply:
`getDocumentComponents` er en funksjon som kalles når vi trykker på knappen, så verdien hentes bare ut da. Men det blir kanskje mest riktig å bruke `getValues` uansett, endrer!